lib/core/decision_service/index.ts

```diff
@@ -835,6 +941,21 @@ export class DecisionService {
         reasons: decideReasons,
       });
     }
+    if (holdout()) {
```

Comment:
Nit: Small improvements for better readability.
```suggestion
    const isEnableHO = holdout()
    if (isEnableHO) {
```

Reply:
I have put `featureToggle.holdout()`  for readability like prev implementation here [code](https://github.com/optimizely/javascript-sdk/blob/master/lib/project_config/project_config.ts#L352).